google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITDMLTest.java

```diff
@@ -76,8 +76,31 @@ public final class ITDMLTest {
   private static final String UPDATE_DML = "UPDATE T SET V = 100 WHERE K LIKE '%d-boo%%';";
   private static final String DELETE_DML = "DELETE FROM T WHERE K like '%d-boo%%';";
 
+  private static final String INSERT_IGNORE_DML =
+      "INSERT OR IGNORE INTO T (k, v) VALUES ('%d-boo1', 1), ('%d-boo2', 2), ('%d-boo3', 3), "
+          + "('%d-boo4', 4), ('%d-boo5', 5);";
+  private static final String INSERT_UPDATE_DML =
+      "INSERT OR UPDATE INTO T (k, v) VALUES ('%d-boo1', 10), ('%d-boo2', 20), ('%d-boo3', 30), "
+          + "('%d-boo4', 40), ('%d-boo5', 50);";
+  private static final String INSERT_IGNORE_DML_POSTGRESQL =
+      "INSERT INTO T (k, v) VALUES ('%d-boo1', 1), ('%d-boo2', 2), ('%d-boo3', 3), "
+          + "('%d-boo4', 4), ('%d-boo5', 5) "
+          + "ON CONFLICT(k) DO NOTHING;";
+  private static final String INSERT_UPDATE_DML_POSTGRESQL =
+      "INSERT INTO T (k, v) VALUES ('%d-boo1', 10), ('%d-boo2', 20), ('%d-boo3', 30), "
+          + "('%d-boo4', 40), ('%d-boo5', 50) "
+          + "ON CONFLICT(k) DO UPDATE SET k = excluded.k, v = excluded.v;";
+  private static final String INSERT_IGNORE_DML_FROM_QUERY =
```

Comment:
Can you pls add `ORDER BY v` to each subquery as below:
`SELECT .. FROM T ORDER BY v`

There should be no change change to the result

Reply:
What is the error due to `ORDER BY` ?